addons/project/__manifest__.py

```diff
@@ -53,7 +53,6 @@
             'project/static/src/js/project_kanban.js',
             'project/static/src/js/project_list.js',
             'project/static/src/js/project_rating_reporting.js',
-            'project/static/src/js/project_name_with_subtask_count_widget.js',
```

Comment:
No need to define the widgets folder in the assets ? 🤔

Reply:
Ok. git history issue see : https://github.com/odoo/odoo/commit/7ed7f414ec2f81c0e8447a72a8a5a9574d16e15a#diff-d4672f45f1188976a4d632a14077092bb708978a8ddbd27b9f90842c35fe36c7R60

Reply:
Oversight in the git history, I cleaned it up.